lib/utils/timestamp.js

```diff
@@ -1,19 +1,30 @@
 const moment = require('moment');
 const {config} = require('../../config');
 const timestampOff = require('../constants/timestamp').none;
+const {curry} = require('ramda');
 
 /**
- * Formats the timestamp for logging purposes
- * @{Date} date - date to format, defaults to current date.
- * @returns {string}
+ * Formats date.
+ * @param {string} format - format string
+ * @param {Date} date - date to format
+ * @returns {string} - if format is timestampOff, returns empty string
  */
-module.exports = {
-	format: (date = undefined) => {
-		const currentTimestamp = config.timestamp;
+const format = curry((format, date) => {
+	if (format === timestampOff)
+		return '';
 
-		if (currentTimestamp === timestampOff)
-			return '';
+	return moment(date).format(format);
+});
 
-		return moment(date).format(currentTimestamp);
-	},
+/**
+ * Formats the date depending on timestamp in config.
+ * For logging purposes.
+ * @param {Date} date - date to format
+ * @returns {string}
+ */
+const formatDate = format(config.timestamp);
+
+module.exports = {
+	format,
+	formatDate,
```

Comment:
So, from what I can see in this file the function that should be used in logs changed from `format` to `formatData` (as this is the one that have format pre-defined). However, I don't see corresponding change in logger... Am I missing something?

Reply:
correct, thats a mistake